templates/blocks/spreadsheettable.twig

```diff
@@ -0,0 +1,31 @@
+{% block spreadsheettable %}
+
```

Comment:
The block shouldn't display anything if no matching file was found or if there is no data. Currently the search box is always displayed.